### PR TITLE
resolve file duplicate name on creating new file

### PR DESCRIPTION
**Pull request by JanAckermann**
## Description
image

## Related Issue

- Fixes https://github.com/owncloud/web/issues/7539

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-files/src/components/AppBar/CreateAndUpload.vue [hunk 1]
```diff
@@ -841,44 +850,53 @@ export default defineComponent({
 #create-list {
   li {
     border: 1px solid transparent;
+
```
[the remark this answers is not in the document]
Comment (JanAckermann) [reply packages/web-app-files/src/components/AppBar/CreateAndUpload.vue:853]: but ok for me

[the remark this answers is not in the document]
Comment (kulmann) [reply packages/web-app-files/src/components/AppBar/CreateAndUpload.vue:853]: Probably some linting on save 😉 Ok for me as well 😇

File: packages/web-runtime/src/services/auth/userManager.ts [hunk 2]
```diff
@@ -181,6 +181,7 @@ export class UserManager extends OidcUserManager {
       await this.clientService.owncloudSdk.users.getUser(login.id),
       await this.clientService.owncloudSdk.users.getUserGroups(login.id)
     ])
+    console.log(userGroups)
```
Comment (kulmann) [inline packages/web-runtime/src/services/auth/userManager.ts:184]: dev leftover

File: changelog/unreleased/enhancement-propose-file-name-while-creating-file [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Enhancement: Propose file name while creating a new file
```
Comment (kulmann) [inline changelog/unreleased/enhancement-propose-file-name-while-creating-file:1]:
```suggestion
Enhancement: Propose unique file name while creating a new file
```

Comment (kulmann) [reply changelog/unreleased/enhancement-propose-file-name-while-creating-file:1]: (if that's not again too long of a title for calens)

File: changelog/unreleased/enhancement-propose-file-name-while-creating-file [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+We've added the possibility while creating a new file or folder to via the 'New' button
```
Comment (kulmann) [inline changelog/unreleased/enhancement-propose-file-name-while-creating-file:3]:
```suggestion
We're proposing unique file names when creating a new file or folder via the 'New' file menu.
```

Comment (kulmann) [reply changelog/unreleased/enhancement-propose-file-name-while-creating-file:3]: (+ drop the following line)

**Review by kulmann — CHANGES_REQUESTED** [review]

**Review by kulmann — APPROVED** [review]
😍

**Pull request merged**